src/t2iapi/operation/operation_requests.proto

```diff
@@ -20,4 +21,16 @@ Request to change the devices state, which results in setting the requested Oper
 message SetOperatingModeRequest {
     string handle = 1;
     OperatingMode operating_mode = 2;
-}
\ No newline at end of file
+}
+
+/*
+Request to set the @InvocationEffectiveTimeout of the pm:AbstractOperationDescriptor with the given handle to a value
+that is less than or equal to the requested threshold.
+*/
+message SetInvocationEffectiveTimeoutLessThanOrEqualToThresholdRequest {
+    string handle = 1;  // handle of the pm:AbstractOperationDescriptor for which the @InvocationEffectiveTimeout
+                        // shall be set
+    google.protobuf.Duration threshold = 2;  // threshold value in seconds at nanosecond resolution for which applies
```

Comment:
What does the "in seconds at nanosecond resolution" mean? Do you only use the nanosecond part of the duration? That's kinda.. odd.